docs/source/tutorial/production.md

```diff
@@ -3,8 +3,104 @@ title: "4. Run your graph in production"
 description: Start here for the Apollo fullstack tutorial
 ---
 
+<h2 id="deploy">Deploy your graph API</h2>
+
+There are several services you can deploy your graph API to such as Heroku, Amazon Lambda, Netlify and Zeit's Now.  We'll deploy our graph API to Zeit's **Now**. Now is a service by Zeit that allows the deployment of an instance of Apollo Server, quickly providing a functional GraphQL endpoint.
+
+You need the following before deployment:
+
+* A [Now](https://zeit.co/now) account.
+* The [Now CLI](https://zeit.co/download#now-cli).
+
+Now looks for a `start` script in the `package.json` file to start the app. Once the script is present, Now will be able to start the app.
+
+Go ahead and run the `now` command from the root directory of the app.
+
+```bash
+$ now
+```
+
+The `now` command immediately deploys your graph API to the cloud and returns the hosted URL. The returned URL comes in this format: `<NOW_APP_NAME>.now.sh`.
+
+You can now query your graph service at `<NOW_APP_NAME>.now.sh/graphql`.
+
+Now also supports direct deployment from a GitHub repository. If your graph API is publicly available on GitHub, you can deploy it via the `now` command like so:
+
+```bash
+now <github-username>/<repository-name>
+```
+
 <h2 id="publish-schema">Publish your schema to Engine</h2>
 
-<h2 id="deploy">Deploy your graph API</h2>
+Safely evolving your schema overtime requires a lot of developer efforts in ensuring that clients consuming your API do not break as a result of schema changes. The Apollo platform provides the ability to publish your schema to Engine via the [Apollo CLI](https://npm.im/apollo).
+
+[Apollo Engine](https://engine.apollographql.com), provides deep insights into your graph layer that enables you to understand, optimize, and control your graph service in production with confidence.
+
+Once you publish your schema to Apollo Engine, it becomes the basis for comparison for validating future schemas and avoiding breaking changes. Therefore, a schema should be re-published to Apollo Engine each time a new schema is deployed.
+
+Go ahead and install the `apollo` CLI via npm:
+
+```bash
+npm install --global apollo
+```
+
+To publish a schema, start your GraphQL server and run the command below:
```

Comment:
Before they publish, they need an Engine API key. Can you please add instructions on how to get that key?

Reply:
I added instructions on how to get the API key on line 53

Reply:
Moved the instructions up a bit!